### PR TITLE
andres' chickensexing.js method implemented

### PR DESCRIPTION
**Pull request by ap102**
I've completed the chickenSexing kata and uncommented the respective test :)

### CODE REVIEW

File: katas/chicken-sexing/andres/chickenSexing.js [hunk 1]
```diff
@@ -1,3 +1,11 @@
 export function correctness(bobsDecisions, expertDecisions) {
-
+    var score = 0;
+    for(var i=0; i<bobsDecisions.length; i++){
```
Comment (J0c4) [inline katas/chicken-sexing/andres/chickenSexing.js:3]: Please add space between `for` and `(`, and for `<` operator. Also, please include space between `)` and `{`

File: katas/chicken-sexing/andres/chickenSexing.js [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+        if (bobsDecisions[i]==expertDecisions[i]) {
```
Comment (J0c4) [inline katas/chicken-sexing/andres/chickenSexing.js:4]: Same here for `==`

File: katas/chicken-sexing/andres/chickenSexing.js (hunk 1 above)
Comment (J0c4) [inline katas/chicken-sexing/andres/chickenSexing.js:3]: A better name for `i` could be something like `index`

File: katas/chicken-sexing/andres/chickenSexing.js [hunk 3]
```diff
@@ -1,3 +1,11 @@
...
+            score += 1;
+        }else if(bobsDecisions[i]=='?'||expertDecisions[i]=='?'){
```
Comment (J0c4) [inline katas/chicken-sexing/andres/chickenSexing.js:6]: Same here on spaces

**Review by J0c4 — CHANGES_REQUESTED** [review]

**Comment by ap102** [comment]
I made the mistake to add the 'square-digits.test.js' file to the commit when I was working at the 'andres/square-digits' branch.
I would like to know if I could close this PR and open another without the 'square-digits.test.js' file.

**Comment by J0c4** [comment]
> I made the mistake to add the 'square-digits.test.js' file to the commit when I was working at the 'andres/square-digits' branch.
> I would like to know if I could close this PR and open another without the 'square-digits.test.js' file.

You can remove the file using `git rm katas/square-digits/andres/squareDigits.test.js`, commit that change and push it as a new commit in this same PR

**Comment by J0c4** [comment]
I reviewed your lates updates, those look good! just fix the problem with the file that is not expected and this should be ready to merge

**Comment by ap102** [comment]
That's good to know!, I'm sending the commit right now 🙂
________________________________
De: Jose Carlos ***@***.***>
Enviado: martes, 13 de julio de 2021 21:44
Para: fjcoding/AT-BC-LAT-01 ***@***.***>
Cc: Andres Pinzon Barragan ***@***.***>; Author ***@***.***>
Asunto: Re: [fjcoding/AT-BC-LAT-01] andres' chickensexing.js method implemented (#13)


I reviewed your lates updates, those look good! just fix the problem with the file that is not expected and this should be ready to merge

**Review by J0c4 — APPROVED** [review]

**Pull request merged**